// [[Reason: Semantically invalid. TSC does not emit .d.ts]] ////

//// [tests/cases/compiler/declarationEmitInvalidReference2.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,7 +1,8 @@
 
 
 //// [declarationEmitInvalidReference2.d.ts]
+/// <reference path="invalid.d.ts" />
 declare var x: number;
 //# sourceMappingURL=declarationEmitInvalidReference2.d.ts.map
 /// [Errors] ////
 
